src/scss/_about.scss

```diff
@@ -0,0 +1,49 @@
+//---------------Break Points---------------//
+//50em (800px)
+//61.25em (980px)
+
+.about {
+  background-color: $color-blue;
+  text-align: center;
+  padding: 4em 0; //64px 0
+}
+
+.about__heading {
```

Comment:
Would you suggest doing this will all my scss files, because I know I do this repeatedly?